packages/arcade/lib/src/helpers/server_helpers.dart

```diff
@@ -22,7 +22,9 @@ void setupProcessSignalWatchers(
     await closeServerExit(server);
   });
 
-  ProcessSignal.sigterm.watch().listen((_) async {
-    await closeServerExit(server);
-  });
+  Platform.isWindows
+      ? const Stream.empty()
+      : ProcessSignal.sigterm.watch().listen((_) async {
+          await closeServerExit(server);
+        });
```

Comment:
Instead of using a ternary, can we instead use a simple if statement to check.

```suggestion
  if (!Platform.isWindows) {
    ProcessSignal.sigterm.watch().listen((_) async {
      await closeServerExit(server);
    });
  }
```

Reply:
Sure. I'll commit the suggested code.